packages/cardano-services/test/Asset/TypeormAssetProvider/fixtures/queries.ts

```diff
@@ -0,0 +1,19 @@
+export const findAssets = `
+  SELECT * from asset LIMIT $1
```

Comment:
Why don't we use typeorm repository or query builder for these queries?

Reply:
Fixed on 41f204f